invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/BookDetails/components/BookMetadata/BookMetadata.js

```diff
@@ -16,24 +16,27 @@ import { RequestNewLoanForm } from './components/RequestNewLoanForm';
 import '../../BookDetails.scss';
 
 export default class BookMetadata extends Component {
+  constructor(props) {
+    super(props);
+    this.bookData = this.props.bookDetails.metadata;
```

Comment:
Since this is inside `BookMetadata` perhaps it's enough to call it `this.data`. What do you think?